recipes/pash/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
 
 about:
   home: http://www.bioinformatics.bbsrc.ac.uk/projects/bismark/
```

Comment:
It's the url which is wrong, it should be: https://genboree.org/pash-supp/
=> there, the licence is not specified